consumer/pdata/common.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // AttributeValueType specifies the type of AttributeValue.
-type AttributeValueType int
+type AttributeValueType int32
```

Comment:
Is this one also needed?

Reply:
I think this is for consistency and ensure that or "enum types" are using int32, less memory than int on 64-bits arch